ci/Dockerfile

```diff
@@ -93,8 +96,8 @@ RUN pyenv install 3.7 \
 RUN /opt/toolchains/zephyr-sdk-${ZSDK_VERSION}/setup.sh -c
 
 # Install python packages
-RUN python3 -m pip install poetry \
-    && python3 -m pip install tox \
-    && python3 -m pip install west \
+RUN python3 -m pip install --no-cache-dir poetry \
```

Comment:
Can you tell us why the new added param is needed?

Reply:
The issue stated, that the cache should be disabled in order to reduce the image size.

Reply:
Please comment on the dockerfile the reason :)